_profiles/Phenotype/0.2-DRAFT.html

```diff
@@ -0,0 +1,343 @@
+  description: 'This Phenotype type specification presents the markup for describing
+    a Phenotype.<h3>Changes from Previous Version</h3><ul><li>isAssociatedWith BioChemEntity
+    switch to recommended</li><li>taxonomicRange switch to recommended</li><li>new
+    property: associatedDisease</li></ul>'
+  version: 0.2-DRAFT
+  version_date: 20200512T203625
+  official_type: http://bioschemas.org/Phenotype
+  full_example: https://github.com/BioSchemas/Specifications/tree/master/Phenotype/
+mapping:
+- property: additionalProperty
+  expected_types:
+  - PropertyValue
+  description: |-
+    A property-value pair representing an additional characteristics of the entitity, e.g. a product feature or another characteristic for which there is no matching property in schema.org.
+
+    Note: Publishers should be aware that applications designed to use specific schema.org properties (e.g. http://schema.org/width, http://schema.org/color, http://schema.org/gtin13, ...) will typically expect such data to be provided using those properties, rather than using the generic property/value mechanism.
+  type: ""
+  type_url: ""
+  bsc_description: Whenever possible, please use a property coined in a third-party
+    well-know vocabulary. For instance, you can directly use http://purl.obolibrary.org/obo/RO_0002327
+    as a property to express how a protein or gene enables some GO molecular function.
+    If you still want or need to use additionalProperty, please use (i) property name
+    to specify the name of the property, (ii) additionalType (if possible) to better
+    specify the nature of the property, and (iii) value to link to the object/range
+    of this property.
+  marginality: Optional
+  cardinality: MANY
+  controlled_vocab: ""
+  example: ""
+- property: alternateName
+  expected_types:
+  - Text
+  description: An alias for the item.
+  type: ""
+  type_url: ""
+  bsc_description: ""
+  marginality: Optional
+  cardinality: MANY
+  controlled_vocab: ""
+  example: |-
+    {
+        "@type": "Phenotype",
+        "alternateName": "abnormal finger morphology"
+    }
+- property: anatomicalLocation
+  expected_types:
+  - DefinedTerm
+  description: Any part of an organism, typically a component of an anatomical system.
+    Organs, tissues, and cells are all anatomical locations.
+  type: bioschemas
+  type_url: anatomicalLocation
+  bsc_description: ""
+  marginality: Optional
+  cardinality: MANY
+  controlled_vocab: 'Any term from a suitable ontology such as [Foundational Model
+    of Anatomy Ontology](http://www.ontobee.org/ontology/FMA), [EMAPA](http://www.ontobee.org/ontology/EMAPA), '
+  example: |-
+    {
+      "@type": "Phenotype",
+      "anatomicalLocation": {
+                    "@type": "DefinedTerm",
+                    "name": "limb digit",
+                    "inDefinedTermSet": "http://purl.obolibrary.org/obo/emapa.owl",
+                    "termCode": "EMAPA_32725"
+            }
+    }
+- property: associatedDisease
```

Comment:
This property does not exist on the corresponding type, see https://bioschemas.org/types/Phenotype/

Reply:
Resolved on BioChemEntity branch of schemaorg repo. Need to copy across to website.

Reply:
Do you mean a branch on https://github.com/schemaorg/schemaorg? I did not find it, could you please provide a URL? thanks

Reply:
Yes, that is the place. I've made the necessary changes